### ISSUE TITLE
 Cannot read properties of null (reading 'tw')

### ISSUE THREAD

**Issue by Geokureli** [deepnight/ldtk#811]
Stack:
```
LDtk version: 1.2.4-64bits
Cannot read properties of null (reading 'tw')
TypeError
TypeError: Cannot read properties of null (reading 'tw')
    at ui_LevelInstanceForm.updateLevelPropsForm (file:///Applications/LDtk.app/Contents/Resources/app.asar/assets/js/renderer.js:249962:106)
    at ui_LevelInstanceForm.useLevel (file:///Applications/LDtk.app/Contents/Resources/app.asar/assets/js/renderer.js:249496:8)
    at ui_LevelInstanceForm.onGlobalEvent (file:///Applications/LDtk.app/Contents/Resources/app.asar/assets/js/renderer.js:249524:9)
    at ui_modal_panel_WorldPanel.onGlobalEvent (file:///Applications/LDtk.app/Contents/Resources/app.asar/assets/js/renderer.js:272082:27)
    at misc_GlobalEventDispatcher.emit (file:///Applications/LDtk.app/Contents/Resources/app.asar/assets/js/renderer.js:202095:4)
    at page_Editor.selectLevel (file:///Applications/LDtk.app/Contents/Resources/app.asar/assets/js/renderer.js:214663:11)
    at WorldTool.onMouseUp (file:///Applications/LDtk.app/Contents/Resources/app.asar/assets/js/renderer.js:22265:21)
    at page_Editor.onMouseUp (file:///Applications/LDtk.app/Contents/Resources/app.asar/assets/js/renderer.js:210802:18)
    at HTMLBodyElement.<anonymous> (file:///Applications/LDtk.app/Contents/Resources/app.asar/assets/js/renderer.js:205004:10)
    at HTMLBodyElement.dispatch (/Applications/LDtk.app/Contents/Resources/app.asar/assets/js/jquery.min.js:2:43064)
```
Processes:
```
#0 App
 |--- #4 page.Editor
       |--- #5 misc.FileWatcher
       |--- #6 display.WorldRender
       |--- #7 display.LevelRender [PAUSED]
       |--- #8 display.Camera
       |--- #9 display.Rulers[--]
       |--- #10 tool.SelectionTool[--]
       |--- #11 tool.lt.DoNothing[--]
       |--- #12 WorldTool
       |--- #13 tool.PanView[--]
       |--- #14 tool.lt.EntityTool[--]
       |--- #15 ui.modal.panel.WorldPanel
       |--- #16 display.EntityRender
       |--- #17 display.EntityRender
       |--- #18 display.EntityRender
       |--- #19 display.EntityRender
       |--- #20 display.EntityRender
       |--- #21 display.EntityRender
       |--- #22 display.EntityRender
       |--- #23 display.EntityRender
       |--- #24 display.EntityRender
       |--- #25 display.EntityRender
       |--- #26 display.EntityRender
       |--- #27 display.EntityRender
       |--- #28 display.EntityRender
       |--- #29 display.EntityRender
       |--- #30 display.EntityRender
       |--- #31 display.EntityRender
       |--- #32 display.EntityRender
       |--- #33 display.EntityRender
```

Log:
```
[RENDER]       Rendering world level data.Level#5aef8d80-66b0-11ec-b3ca-a30ee346445f "m_filler_left_1"...
[RENDER]       Rendering world level data.Level#5af4bda0-66b0-11ec-b3ca-2579c4e7294e "m_filler_right_1"...
[RENDER]       Rendering world level data.Level#5afaff30-66b0-11ec-b3ca-11232568dd3c "m_left_right_1"...
[RENDER]       Rendering world level data.Level#5b018ee0-66b0-11ec-b3ca-09270bb03d7c "m_right_left_1"...
[RENDER]       Rendering world level data.Level#5b075b40-66b0-11ec-b3ca-091da749d64f "m_topside_end_1"...
[RENDER]       Rendering world level data.Level#5b0d9cd0-66b0-11ec-b3ca-893b01eb9ccb "m_topside_filler_1"...
[RENDER]       Rendering world level data.Level#5d25f580-66b0-11ec-b3ca-bf655e12b247 "desertBonus1_2"...
[RENDER]       Rendering world level data.Level#83b9aef0-2a00-11ed-9c4e-b7f9b59de922 "excavation_start"...
[CACHE]        Caching image ../../../../../../OneDrive/Pictures/guide-bg.png...
[CACHE]         -> identified as Unknown
[ERROR]        TypeError: Cannot read properties of null (reading 'length')
[RENDER]       Rendering world level data.Level#09844000-2a00-11ed-aa84-b1de6bd8bf4a "excavation_center"...
[CACHE]        Caching image ../../../../../../OneDrive/Pictures/guide-bg.png...
[CACHE]         -> identified as Unknown
[ERROR]        TypeError: Cannot read properties of null (reading 'length')
[RENDER]       Rendering world level data.Level#6c0c77c0-2a00-11ed-aa84-b10f4aca99a8 "excavation_underground_entrance"...
[CACHE]        Caching image ../../../../../../OneDrive/Pictures/guide-bg.png...
[CACHE]         -> identified as Unknown
[ERROR]        TypeError: Cannot read properties of null (reading 'length')
[RENDER]       Rendering world level data.Level#e87a58e0-5110-11ed-a427-6167eb5291a1 "excavation_underground"...
[CACHE]        Caching image ../../../../../../OneDrive/Pictures/guide-bg.png...
[CACHE]         -> identified as Unknown
[ERROR]        TypeError: Cannot read properties of null (reading 'length')
[RENDER]       Rendering world level data.Level#b975e040-5110-11ed-8416-c39be6abcead "excavation_sand"...
[RENDER]       Rendering world level data.Level#9b11d450-5110-11ed-b95d-0deca8add1c0 "excavation_tower_room_A"...
[RENDER]       Rendering world level data.Level#2f54edd0-5110-11ed-b95d-73f99a74e33a "excavation_tower_room_B"...
[RENDER]       Rendering world level data.Level#d37292e0-5110-11ed-b95d-bb60dc1eaded "excavation_tower_left"...
[EVENT]        LevelSelected 749
[USERACTION]   Opened level data.Level#83b9aef0-2a00-11ed-9c4e-b7f9b59de922 "excavation_start"
[CACHE]        Caching image ../../../../../../OneDrive/Pictures/guide-bg.png...
[CACHE]         -> identified as Unknown
[ERROR]        TypeError: Cannot read properties of null (reading 'length')
[WARNING]      Full level render requested
[CACHE]        Caching image ../../../../../../OneDrive/Pictures/guide-bg.png...
[CACHE]         -> identified as Unknown
[ERROR]        TypeError: Cannot read properties of null (reading 'length')
[CACHE]        Caching image ../../../../../../OneDrive/Pictures/guide-bg.png...
[CACHE]         -> identified as Unknown
[ERROR]        TypeError: Cannot read properties of null (reading 'length')
[GENERAL]      Page started: page.CrashReport()
[FILE]         Loading page template: crashReport from /Applications/LDtk.app/Contents/Resources/app.asar/assets/tpl/pages/crashReport.html
[ERROR]        Cannot read properties of null (reading 'tw') (TypeError)
[ERROR]        TypeError: Cannot read properties of null (reading 'tw')
    at ui_LevelInstanceForm.updateLevelPropsForm (file:///Applications/LDtk.app/Contents/Resources/app.asar/assets/js/renderer.js:249962:106)
    at ui_LevelInstanceForm.useLevel (file:///Applications/LDtk.app/Contents/Resources/app.asar/assets/js/renderer.js:249496:8)
    at ui_LevelInstanceForm.onGlobalEvent (file:///Applications/LDtk.app/Contents/Resources/app.asar/assets/js/renderer.js:249524:9)
    at ui_modal_panel_WorldPanel.onGlobalEvent (file:///Applications/LDtk.app/Contents/Resources/app.asar/assets/js/renderer.js:272082:27)
    at misc_GlobalEventDispatcher.emit (file:///Applications/LDtk.app/Contents/Resources/app.asar/assets/js/renderer.js:202095:4)
    at page_Editor.selectLevel (file:///Applications/LDtk.app/Contents/Resources/app.asar/assets/js/renderer.js:214663:11)
    at WorldTool.onMouseUp (file:///Applications/LDtk.app/Contents/Resources/app.asar/assets/js/renderer.js:22265:21)
    at page_Editor.onMouseUp (file:///Applications/LDtk.app/Contents/Resources/app.asar/assets/js/renderer.js:210802:18)
    at HTMLBodyElement.<anonymous> (file:///Applications/LDtk.app/Contents/Resources/app.asar/assets/js/renderer.js:205004:10)
    at HTMLBodyElement.dispatch (/Applications/LDtk.app/Contents/Resources/app.asar/assets/js/jquery.min.js:2:43064)

[GENERAL]      
#0 App
 |--- #4 page.Editor
       |--- #5 misc.FileWatcher
       |--- #6 display.WorldRender
       |--- #7 display.LevelRender [PAUSED]
       |--- #8 display.Camera
       |--- #9 display.Rulers[--]
       |--- #10 tool.SelectionTool[--]
       |--- #11 tool.lt.DoNothing[--]
       |--- #12 WorldTool
       |--- #13 tool.PanView[--]
       |--- #14 tool.lt.EntityTool[--]
       |--- #15 ui.modal.panel.WorldPanel
       |--- #16 display.EntityRender
       |--- #17 display.EntityRender
       |--- #18 display.EntityRender
       |--- #19 display.EntityRender
       |--- #20 display.EntityRender
       |--- #21 display.EntityRender
       |--- #22 display.EntityRender
       |--- #23 display.EntityRender
       |--- #24 display.EntityRender
       |--- #25 display.EntityRender
       |--- #26 display.EntityRender
       |--- #27 display.EntityRender
       |--- #28 display.EntityRender
       |--- #29 display.EntityRender
       |--- #30 display.EntityRender
       |--- #31 display.EntityRender
       |--- #32 display.EntityRender
       |--- #33 display.EntityRender
 |--- #34 page.CrashReport

[EVENT]        LevelSelected 749
[USERACTION]   Opened level data.Level#83b9aef0-2a00-11ed-9c4e-b7f9b59de922 "excavation_start"
[TIMELINE]     Garbage collected level: #834
[FILE]         Cleared all file watches
```

**Comment by deepnight** [comment]
Thanks, this should be fixed in 1.2.5

**Comment by Geokureli** [comment]
Thanks for the extremely fast turnaround on all of my issues
